### PR TITLE
Update Blazor components limitations

### PR DESCRIPTION
**Pull request by wnvko**
Update Blazor components limitations.
Closes #15657

_Note: in known issues we have now:_
 - _cannot generate - this means component supports the feature, but it is not generated, e.g. divider in card._
 - _does not support - this means component does not support this feature, e.g. date picker in dialog mode._

### CODE REVIEW

File: en/blazor-support.md [hunk 1]
```diff
@@ -52,12 +52,38 @@ Generating Blazor application has the following limitations:
...
-- Radio group does not support horizontal mode.
-- Grid cannot generate a toolbar. This disallows the generation of advanced filtering, exporting, pinning, and hiding. Setting cell and row selection do not work simultaneously. Column selection, paging, grouping, and column moving are not supported yet. Filtering is working only in excel style mode.
-- For all components toggle actions are not generated yet.
+- Card:
+    - cannot generate reversed actions.
+    - cannot generate divider in actions column card type.
+- Navigation Bar - cannot generate title.
+- Navigation Drawer - does not support pin state and pin threshold.
+- Avatar:
+    - does not use IgbIcon in generated code.
```
Comment (damyanpetev) [inline en/blazor-support.md:61]: How about we put a single global note about all icons using simple Material Icons font content instead of a full `IgbIcon` instead of putting this note on every component that uses icons?

Also is this the reason why the Avatar is marked as partial in the supported components table? I'm inclined to mark it as fully functional actually, there's nothing wrong with the Avatar itself and it works just fine for the features we expose, right? It's the icon inside, which is global and separate, sort of.

PS: Also do mark code references such as component classes with `` ` ``s if you're going to use the names like that

File: en/blazor-support.md [hunk 2]
```diff
@@ -52,12 +52,38 @@ Generating Blazor application has the following limitations:
...
+    - initials type shows entire contend instead of first two chars.
+- Icon - does not use IgbIcon in generated code.
+- Date Picker:
+    - does not support dialog mode.
+    - does not support months.
+    - does not support action buttons.
+- Floating Action Button - does not use IgbIcon in generated code.
+- Icon Button - does not use IgbIcon in generated code.
```
Comment (damyanpetev) [inline en/blazor-support.md:69]: The issue of the Icon button is not the that it doesn't use the icon, but that it's not an actual icon button in Blazor, but a basic button with icon inside. It's the same underlying issue we have with icons that causes this, but still the description can be more accurate

File: en/generate-app/generate-app-overview.md [hunk 3]
```diff
@@ -33,49 +33,49 @@ Right now AppBuilder generates application for Angular and Blazor. Below is a li
 |------------------------|--------------------|--------------------|
 | **Layouts**                                                      |
 | Absolute Layout        | :heavy_check_mark: | :heavy_check_mark: |
-| Card                   | :heavy_check_mark: | :question:         |
+| Card                   | :heavy_check_mark: | :warning:          |
```
Comment (damyanpetev) [inline en/generate-app/generate-app-overview.md:36]: Still doesn't look quite right, a warning doesn't convey "available, but limited", I was going to suggest a yellow tick with asterisk `✔*` or something similar, but it can't be colored, so alternatively perhaps - :heavy_check_mark::construction:?

PS: Also we should totally consider the use of those emoji since they differ per platform and frankly look terrible on Android for example:
image

**Review by damyanpetev — COMMENTED** [review]
I actually have a bit of issue with the entire structure of this:
Having this info split in two topics is incoherent for one;
The table with supported components has no place in the Generate App topic (hint: it's valid for code preview too..) and same goes for the Platform/Framework switch info.
Also having a lone Blazor support topic makes it look that is the only generation available since there's no equivalent for Angular. And the content has a lot of duplicate fluff.
// CC @zdrawku @ig-georgeA 

IMO this info should be pulled into it's own topic and/or section (Code Generation > Switching platforms, Code Generation > Supported platforms, whatever) where we can list what is supported per platform and the limitations for Blazor. Kinda wondering if it'd be too heavy if the per-component limitations are inside the table actually

File: en/generate-app/generate-app-overview.md (hunk 3 above)
Comment (wnvko) [reply en/generate-app/generate-app-overview.md:36]: [Here ](https://infragisticsinc297-my.sharepoint.com/:w:/g/personal/mvenkov_infragistics_com/EbgnJjx6_HBKhc7MAxVaNqgB3aZMjPW65cr1yI7RNrJpgQ?e=XRLR1j)is document with all the limitations I've found. Not all of these are put in the documentation.

**Comment by zdrawku** [comment]
> The table with supported components has no place in the Generate App topic (hint: it's valid for code preview too..) and same goes for the Platform/Framework switch info.

Having a separate Blazot support topic was a marketing request in order to stress on it.

I agree, we should think of a way to restructure the docs a bit. @damyanpetev @ig-georgeA lets discuss that on the Scrum today

Example:
image

**Comment by Lipata** [comment]
@wnvko you can skip the structural changes and apply other comments and we will merge this PR.

File: en/blazor-support.md (hunk 1 above)
Comment (wnvko) [reply en/blazor-support.md:61]: > Also is this the reason why the Avatar is marked as partial in the supported components table? I'm inclined to mark it as fully functional actually, there's nothing wrong with the Avatar itself and it works just fine for the features we expose, right? 

When set a string with length bigger than two letters in Blazor Avatar it is trying to show entire string instead of just first two letters. In AppBuilde and in Angular generated application only first two letters are shown.

File: en/blazor-support.md [hunk 4]
```diff
@@ -52,12 +52,35 @@ Generating Blazor application has the following limitations:
...
-- Navigation drawer works only in the overlay state. The mini version does not work too.
-- Date picker works only in dropdown mode.
-- The input group does not support date type.
-- Radio group does not support horizontal mode.
-- Grid cannot generate a toolbar. This disallows the generation of advanced filtering, exporting, pinning, and hiding. Setting cell and row selection do not work simultaneously. Column selection, paging, grouping, and column moving are not supported yet. Filtering is working only in excel style mode.
-- For all components toggle actions are not generated yet.
+- Card:
+    - cannot generate reversed actions.
+    - cannot generate divider in actions column card type.
+- Navigation Bar - cannot generate title.
```
Comment (damyanpetev) [inline en/blazor-support.md:58]: This is actually not true? A simple title does generate just fine from what I've tested. Other content in the 'middle' section though doesn't

File: en/blazor-support.md [hunk 5]
```diff
@@ -52,12 +52,35 @@ Generating Blazor application has the following limitations:
 - Bootstrap is turned off in the generated application for now.
 
 Generated components have these limitations:
```
Comment (damyanpetev) [inline en/blazor-support.md:54]:
```suggestion
Generated components have the following limitations:

```

File: en/blazor-support.md [hunk 6]
```diff
@@ -52,12 +52,35 @@ Generating Blazor application has the following limitations:
...
 
 Generated components have these limitations:
-- Navigation drawer works only in the overlay state. The mini version does not work too.
-- Date picker works only in dropdown mode.
-- The input group does not support date type.
-- Radio group does not support horizontal mode.
-- Grid cannot generate a toolbar. This disallows the generation of advanced filtering, exporting, pinning, and hiding. Setting cell and row selection do not work simultaneously. Column selection, paging, grouping, and column moving are not supported yet. Filtering is working only in excel style mode.
-- For all components toggle actions are not generated yet.
+- Card:
+    - cannot generate reversed actions.
```
Comment (damyanpetev) [inline en/blazor-support.md:56]: Hmm, not a huge fan of the "cannot generate" language - it's not can't, more like we currently don't. Perhaps a "not reflected in code" or something similar?

File: en/blazor-support.md [hunk 7]
```diff
@@ -52,12 +52,35 @@ Generating Blazor application has the following limitations:
...
+    - cannot generate divider in actions column card type.
+- Navigation Bar - cannot generate title.
+- Navigation Drawer - does not support pin state and pin threshold.
+- Avatar:
+    - initials type shows entire contend instead of first two chars.
```
Comment (damyanpetev) [inline en/blazor-support.md:61]: 👀 How is that relevant to the code generation? Sounds like a component issue to me, if one at all, might be intended too

File: en/blazor-support.md (hunk 4 above)
Comment (wnvko) [reply en/blazor-support.md:59]: actually, when type is set to Content+Action title is not generated, as it is not send to Blazor.
I've updated the issue

**Review by damyanpetev — APPROVED** [review]

**Review by Lipata — APPROVED** [review]
